core/src/main/scala/org/apache/spark/status/api/v1/OneApplicationResource.scala

```diff
@@ -75,6 +75,46 @@ private[v1] class AbstractApplicationResource extends BaseAppResource {
     }
   }
 
+  @GET
+  @Path("executors/{executorId}/heaps")
```

Comment:
```suggestion
  @Path("executors/{executorId}/heapdump")
```
`/heapdump` or `/heap` looks better to me. 
There is no one-to-one match here with `/threads`

Reply:
Modified